docs/pages/operations-guide.md

```diff
@@ -797,7 +797,7 @@ If you are running DC/OS 1.9 or older, or a version of the service that is older
 
 1. Stop the service. From the DC/OS CLI, enter `dcos package uninstall --app-id=<instancename> <packagename>`.
    For example, `dcos package uninstall --app-id=kafka-dev confluent-kafka`.
-1. Clean up remaining reserved resources with the framework cleaner script, `janitor.py`. See [DC/OS documentation](https://docs.mesosphere.com/1.9/deploying-services/uninstall/#framework-cleaner) for more information about the framework cleaner script.
+1. Clean up remaining reserved resources with the `janitor.py` script. See [DC/OS documentation](https://docs.mesosphere.com/1.10/deploying-services/uninstall/#framework-cleaner) for more information about the cleaner script.
```

Comment:
I thought we didn't want users to run janitor on 1.10 clusters?

Reply:
Yeah, super dont

Reply:
This section seems to be for older versions though?

> Older versions
> If you are running DC/OS 1.9 or older, or a version of the service that is older than 2.0.0-x, follow these steps:
> 1. Stop the service.
> ...

Reply:
That is correct, @mpereira—this is the section for DC/OS versions older than 1.10.

Reply:
There may, of course, be other stale content in the doc.